server/webserver.go

```
@@ -76,6 +76,18 @@ func (s *LivepeerServer) cliWebServerHandlers(bindAddr string) *http.ServeMux {
 	mux := http.DefaultServeMux
 	http.DefaultServeMux = http.NewServeMux()
 
+	mux.HandleFunc("/signMessage", func(w http.ResponseWriter, r *http.Request) {
+		if err := r.ParseForm(); err != nil {
+			glog.Errorf("Parse Form Error: %v", err)
+		}
+		message := r.FormValue("message")
+		signed, err := s.LivepeerNode.Eth.Sign([]byte(message))
+		if err != nil {
+			glog.Errorf("error: %v", err)
```

Let's return an HTTP error here - [example](https://github.com/livepeer/go-livepeer/blob/a9e6dd764ee179db69309618917d176b5cba26fc/server/handlers.go#L110)